src/main/java/com/gregtechceu/gtceu/api/capability/recipe/ItemRecipeCapability.java

```diff
@@ -553,7 +553,7 @@ public void applyWidgetInfo(@NotNull Widget widget,
                         tooltips.add(Component.translatable("gtceu.gui.content.per_tick"));
                     }
                 });
-                if (io == IO.IN && this.of(content.content) instanceof IntCircuitIngredient) {
+                if (io == IO.IN && this.of(content.content) instanceof IntCircuitIngredient || content.chance == 0) {
```

Comment:
Can you add parentheses to make sure it only happens on `IO.IN`
Also switch the order of the last two expressions so that we don't have to do an unnecessary type check if the chance is 0.
```suggestion
                if (io == IO.IN && (content.chance == 0 || this.of(content.content) instanceof IntCircuitIngredient)) {
```